examples/cstg/html/index.html

```diff
@@ -14,89 +14,52 @@
         serverPublicKey: '{{ SERVER_PUBLIC_KEY }}',
       };
 
-      let callbackCounter = {
```

Comment:
Should we remove `<script defer src="{{ EUID_JS_SDK_URL }}"></script>` from this file?